plugins/woocommerce/src/Admin/PluginsInstallLoggers/AsyncPluginsInstallLogger.php

```diff
@@ -223,7 +223,18 @@ function( $extension ) {
 				continue;
 			}
 
-			$track_data[ 'install_time_' . $this->get_plugin_track_key( $plugin ) ] = $this->get_timeframe( $data['time'][ $plugin ] );
+			$plugin_track_key                                  = $this->get_plugin_track_key( $plugin );
+			$install_time                                      = $this->get_timeframe( $data['time'][ $plugin ] );
+			$track_data[ 'install_time_' . $plugin_track_key ] = $install_time;
+
+			wc_admin_record_tracks_event(
+				'coreprofiler_store_extension_installed_and_activated',
+				array(
+					'success'             => true,
```

Comment:
Can we track async failures as well?

Reply:
Yes, I think we can add a track in [add_error](https://github.com/woocommerce/woocommerce/blob/trunk/plugins/woocommerce/src/Admin/PluginsInstallLoggers/AsyncPluginsInstallLogger.php#L125-L133). 👍 

@moon0326 Could you confirm if we need it and the props format:


```
{
    success: boolean,
    installed_extension: string;
    install_time?: string;
    error_message?: string;
}
```

Reply:
I've added it in 5f3388e1dc09a47c7026d89a8aee89fd6e181fbc. Also updated the test instructions.